src/main/java/org/folio/search/service/setter/instance/Date1FieldProcessor.java

```diff
@@ -0,0 +1,34 @@
+package org.folio.search.service.setter.instance;
+
+import java.util.regex.Pattern;
+import org.apache.commons.lang3.StringUtils;
+import org.folio.search.domain.dto.Dates;
+import org.folio.search.domain.dto.Instance;
+import org.folio.search.service.setter.FieldProcessor;
+import org.springframework.stereotype.Component;
+
+@Component
+public class Date1FieldProcessor implements FieldProcessor<Instance, String> {
+
+  private static final Pattern FOUR_DIGIT_REGEX = Pattern.compile("^\\d{4}$");
```

Comment:
It could be less than 4 numbers, like 998, 503, 23, 1

Reply:
updated